### PR TITLE
Added Water Wave Effect

### PR DESCRIPTION
**Pull request by devarshidoshi**
Added Water Wave Effect By Me

### CODE REVIEW

**Comment by devarshidoshi** [comment]
Please Approve My Pull Request If You Love It

File: animations/Water.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
+
+@import "compass/css3";
```
Comment (NiallEccles) [inline animations/Water.css:2]: I think you need to remove this. Is there any reason why you need this?

File: animations/Water.css [hunk 2]
```diff
@@ -0,0 +1,48 @@
+
+@import "compass/css3";
+
+.Water {
+  text-transform: uppercase;
+  font-family: 'Cabin Condensed', sans-serif;
```
Comment (NiallEccles) [inline animations/Water.css:6]: This font won't be available to those who do not have it installed on their machine. Could you use a Google Font? or a system font? That way anyone can view it.

**Review by NiallEccles — CHANGES_REQUESTED** [review]

File: animations/Water.css (hunk 2 above)
Comment (devarshidoshi) [reply animations/Water.css:6]: Thanks for your response.. I have made the desired changes Thankk Youu

**Comment by NiallEccles** [comment]
Nice! Great animation.

**Comment by devarshidoshi** [comment]
Thankk Youu So Much!!

**Pull request merged**